es.go

```diff
@@ -1630,6 +1630,21 @@ type AllocateStalePrimary struct {
 	AcceptDataLoss bool `json:"accept_data_loss,omitempty"`
 }
 
+// Reroute allows to change the allocation of individual shards in the cluster.
```

Comment:
What if we called it `RerouteWithRetryFailed` and make the comment read `RerouteWithRetryFailed retries allocation of shards that are blocked due to too many subsequent allocation failures.`? what do you think?

Reply:
Make sense. I will update this 🙂